source-map-uploader-plugin.js

```diff
@@ -44,8 +44,18 @@ class BugsnagSourceMapUploaderPlugin {
           // for each *.map file, find a corresponding source file in the chunk
           const source = map ? chunk.files.find(file => file === map.replace('.map', '')) : null
 
-          if (!source || !map) {
-            console.warn(`${LOG_PREFIX} no source/map pair found for chunk "${chunk.id}"`)
+          if (!source) {
+            console.warn(`${LOG_PREFIX} no corresponding source found for "${map}" in chunk "${chunk.id}"`)
```

Comment:
If `map` is not defined, `source` will be `null`, making this warning flag up.  Is there a way to make `map` here more descriptive for that case (rather than `undefined`?)

Reply:
`map` will always be defined, so I removed the ternary above too.